.travis/playbook.yml

```diff
@@ -28,7 +29,6 @@
       secret_key: 'secret'
       databases:
         default:
-          ENGINE: "{{ pulp_db_backend }}"
```

Comment:
Where does the definition of ENGINE come from then, a defaults.yaml or something?

Reply:
Yup: https://github.com/pulp/ansible-pulp/blob/master/roles/pulp-database/defaults/main.yml#L9

Though, since we no longer allow maria, its probably never going to be changed.